### DISCUSSION TITLE
install php 8.0 on mac (Error!)

### DISCUSSION THREAD

**Discussion by seyedrezabazyar** [shivammathur/homebrew-php#316] [Q&A / Help] [answered]
**Hello**
**I want to install php 8 on my Apple MacBook Pro. (‌Big Sur 11.3.1)**

**My php version is 7**

`php -v`
PHP 7.3.24

`brew upgrade shivammathur/php/php@8.0`
Error: shivammathur/php/php not installed

**The version shows something that is not installed... 🤦** 

**Let's go to the PHP 8.0 installation** 

`brew install shivammathur/php/php@8.0`
Successfully installed

**link it:**

`brew link --overwrite --force php@8.0`
Warning: Already linked: /usr/local/Cellar/php/8.0.6

**unlink:**

`brew unlink php`
Unlinking /usr/local/Cellar/php/8.0.6... 24 symlinks removed.

**Re-linking version 8.0:**

`brew link --overwrite --force php@8.0`
Linking /usr/local/Cellar/php/8.0.6... 24 symlinks created.

`php -v`
WARNING: PHP is not recommended
PHP is included in macOS for compatibility with legacy software.
Future versions of macOS will not include PHP.
PHP 7.3.24-(to be removed in future macOS) (cli) (built: Feb 28 2021 09:53:14) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.3.24, Copyright (c) 1998-2018 Zend Technologies

**What can I do to link version 8 and replace version 7?**

**Comment by shivammathur** [comment, marked as answer]
@seyedrezabazyar 
After linking, restart your terminal, and then test.

**Reply by seyedrezabazyar** [reply]
Thanks 👍 
I have downloaded version 8.0. But it links to version 8.0.6. Does this not cause a problem?

**Reply by shivammathur** [reply]
8.0.6 is the latest patch release for PHP 8.0.
Ref: https://www.php.net/archive/2021.php#2021-05-06-2
